kyo-prelude/shared/src/main/scala/kyo2/internal/BaseKyoTest.scala

```diff
@@ -39,6 +39,15 @@ trait BaseKyoTest[S]:
     given eitherCanEqual[T, U]: CanEqual[Either[T, U], Either[T, U]] = CanEqual.derived
     given throwableCanEqual: CanEqual[Throwable, Throwable]          = CanEqual.derived
 
+    def untilTrue[S](f: => Boolean < S): Boolean < S =
```

Comment:
Are you sure this should be by name?

Reply:
Yes, some tests perform side effects in the thunk